airflow/providers/amazon/aws/transfers/glacier_to_gcs.py

```diff
@@ -0,0 +1,162 @@
+    ):
+        super().__init__(**kwargs)
+        self.aws_conn_id = aws_conn_id
+        self.vault_name = vault_name
+        self.job_id = job_id
+
+    def execute(self, context):
+        hook = GlacierHook(aws_conn_id="aws_default")
+        response = hook.retrieve_inventory_results(vault_name=self.vault_name, job_id=self.job_id)
+        return response
+
+
+class GlacierTransferDataToGCS(BaseOperator):
+    """
+    This operator transfers data from Glacier to Google Cloud Storage
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GlacierTransferDataToGCS`
+
+    :param vault_name: the Glacier vault on which job is executed
+    :type vault_name: string
+    :param bucket_name: the Google Cloud Storage bucket where the data will be transferred
+    :type bucket_name: str
+    :param object_name: the name of the object to check in the Google cloud
+        storage bucket.
+    :type object_name: str
+    :param gzip: option to compress local file or file data for upload
+    :type gzip: bool
+    :param google_impersonation_chain: Optional Google service account to impersonate using
+        short-term credentials, or chained list of accounts required to get the access_token
+        of the last account in the list, which will be impersonated in the request.
+        If set as a string, the account must grant the originating account
+        the Service Account Token Creator IAM role.
+        If set as a sequence, the identities from the list must grant
+        Service Account Token Creator IAM role to the directly preceding identity, with first
+        account from the list granting this role to the originating account (templated).
+    :type google_impersonation_chain: Union[str, Sequence[str]]
+    """
+
+    template_fields = ("vault_name", "bucket_name", "object_name")
+
+    @apply_defaults
+    def __init__(
+        self,
+        *,
+        aws_conn_id="aws_default",
+        vault_name: str,
+        gcp_conn_id="google_cloud_default",
+        delegate_to=None,
+        google_impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        bucket_name: str,
+        object_name: str,
+        gzip: bool,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.aws_conn_id = aws_conn_id
+        self.vault_name = vault_name
+        self.gcp_conn_id = gcp_conn_id
+        self.delegate_to = delegate_to
+        self.impersonation_chain = google_impersonation_chain
+        self.bucket_name = bucket_name
+        self.object_name = object_name
+        self.gzip = gzip
+
+    def execute(self, context):
+        glacier_hook = GlacierHook(aws_conn_id=self.aws_conn_id)
+        gcs_hook = GCSHook(
+            gcp_conn_id=self.gcp_conn_id,
+            delegate_to=self.delegate_to,
+            impersonation_chain=self.impersonation_chain,
+        )
+
+        # todo: should I check if vault exists?
```

Comment:
If the operator fails with a meaningful error then I don't see a reason for it

Reply:
In my opinion errors messages are meaningful and easy to read.
`ERROR - An error occurred (ResourceNotFoundException) when calling the InitiateJob operation: Vault not found for ARN: arn:aws:glacier:eu-central-1:366280393233:vaults/airfloww`

Reply:
Looks good, let's remove this comment

Reply:
Done